### PR TITLE
Feature/new brca pseudonym generator

### PR DESCRIPTION
**Pull request by izcram**
This is a reimplementation of brca_pseudonym_generator.py script. The goal was to remove the dependency of the now obsolete `pyhgvs` library in favor of the biocommons HGVS library as well as supporting an extended gene set beyond BRCA1/2.

Additionally, follow changes were made
* removal of hg36 related columns in build.tsv
* introduction of a `common` package for pipeline code shared across different scripts
* updated UTA version and biocommons HGVS library
* tweaks in Splicing pipeline to ignore genes other than BRCA1/2 (may require rerelease of pipeline)

Data verification is still pending.

### CODE REVIEW

File: pipeline/data_merging/brca_pseudonym_generator.py [hunk 1]
```diff
@@ -1,272 +1,227 @@
...
+                                                  src_ac=vc.ac,
+                                                  dst_ac=dst, dst_alt_aln_method=method)
+
+                    vp = pj.project_variant_forward(vc)
+                    synonyms.append(vp)
+                    vp_norm = hgvs_proc.normalizing(vp)
+                    if vp_norm:
+                        if vp_norm not in synonyms:
+                            logging.info("Found new synonym! " + str(vp_norm) + " " + str(vp) + " " + str(x[PYHGVS_GENOMIC_COORDINATE_38_COL]))
+                        synonyms.append(vp_norm)
```
Comment (zfisch) [inline pipeline/data_merging/brca_pseudonym_generator.py]: cool

File: pipeline/common/test_hgvs_utils.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return (variant_utils.VCFVariant.from_str('chr13:g.32316477:AAG>A').
+            to_hgvs_obj(wrapper.contig_maps[hgvs_utils.HgvsWrapper.GRCh38_Assem]))
+
+
+def test_to_cdna(wrapper, hgvs_variant):
+    cdna = wrapper.to_cdna(hgvs_variant)
+    assert str(cdna) == 'NM 00005.3:c.22_23delAG'
+
+
+def test_to_cdna(wrapper, hgvs_variant):
```
[the remark this answers is not in the document]
Comment (izcram) [reply pipeline/common/test_hgvs_utils.py]: yes, indeed! good catch, also the above test was not running (and would actually fail). This is fixed

File: pipeline/common/hgvs_utils.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+        self.hgvs_dp = hgvs.dataproviders.uta.connect()
+        logging.info("Using UTA instance at {}".format(self.hgvs_dp.url))
+
+        self.hgvs_parser = hgvs.parser.Parser()
+        self.hgvs_norm = hgvs.normalizer.Normalizer(self.hgvs_dp)
+
+        assemblies = [self.GRCh37_Assem, self.GRCh38_Assem]
+
+        self.hgvs_ams = { a : (hgvs.assemblymapper.AssemblyMapper(self.hgvs_dp, assembly_name=a, normalize=False, prevalidation_level=None)) for a in assemblies }
```
[the remark this answers is not in the document]
Comment (izcram) [reply pipeline/common/hgvs_utils.py]: historical reasons ;) I've removed it.

**Comment by zfisch** [comment]
Lookin good! Thanks Marc, merge when ready :)

**Pull request merged**